docs/user-guide/widgets.md

```diff
@@ -90,13 +112,33 @@ Just below the chart's preview, the user is allowed to set:
 
 * The widget **Description**
 
-!!! note
+!!! Note
     None of these options are mandatory, you can add the widget to the map without filling in these fields.
 
 An example of chart widget could be:
 
 <img src="../img/widgets/chart-ex.jpg" class="ms-docimage"/>
 
+The **Chart toolbar**, displayed in the right corner of the chart allows the user to:
+
+<img src="../img/widgets/bar_charts.jpg" class="ms-docimage"/>
+
+* **Download** the chart as a `png` through the <img src="../img/button/download_png.jpg" class="ms-docbutton"/> button.
+
+* **Zoom** the chart through the <img src="../img/button/zoom_chart.jpg" class="ms-docbutton"/> button.
+
+* **Pan** the chart through the <img src="../img/button/pan_chart.jpg" class="ms-docbutton"/> button.
+
+* **Zoom in** the chart through the <img src="../img/button/zoom_in_chart.jpg" class="ms-docbutton"/> button.
+
+* **Zoom out** the chart through the <img src="../img/button/zoom_out_chart.jpg" class="ms-docbutton"/> button.
+
+* **Autoscale** to center the chart in the panel through the <img src="../img/button/autoscale_chart.jpg" class="ms-docbutton"/> button.
+
+* **Reset axes** to return the chart to its original size through the <img src="../img/button/reset_axes_chart.jpg" class="ms-docbutton"/> button.
```

Comment:
* **Reset axes** to return the chart to its initial state through the <img src="../img/button/reset_axes_chart.jpg" class="ms-docbutton"/> button.